app/routes/goal_routes.py

```diff
@@ -1 +1,68 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, make_response, Response
+from app.models.goal import Goal
+from app.models.task import Task
+from .. import db
+from .route_utilities import validate_model, create_model, get_models_with_filters
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+    return create_model(Goal,request_body)
```

Comment:
Nitpick: missing whitespace after comma
```suggestion
    return create_model(Goal, request_body)
```

Reply:
Fixed as requested.